### ISSUE TITLE
Missing batch norm in ResNet-18 weights

### ISSUE THREAD

**Issue by ondrejba** [facebook/fb.resnet.torch#191]
While converting the ResNet-18 pre-trained weights to Tensorflow, I have noticed that some batch normalization layers are missing.

Specifically, when a convolution is applied to a residual connection in order to reduce its spatial resolution, there is no batch normalization after the convolution.

I.e.
```
main branch: Conv->BN->ReLU->Conv->BN
skip branch: Conv (missing BN)
output: (main branch + skip branch)->ReLU
```

**Comment by ondrejba** [comment]
I've found an issue that deals with the exact same thing just now (https://github.com/facebook/fb.resnet.torch/issues/49). I'm sorry for opening a new issue for this.
